src/main/java/xyz/wagyourtail/jsmacros/client/mixins/events/MixinClientPlayerEntity.java

```diff
@@ -80,15 +92,30 @@ public void onOpenEditSignScreen(SignBlockEntity sign, CallbackInfo info) {
             final SignEditScreen signScreen = new SignEditScreen(sign);
             client.openScreen(signScreen);
             for (int i = 0; i < 4; ++i) {
-                ((ISignEditScreen)signScreen).jsmacros_setLine(i, lines.get(i));
+                ((ISignEditScreen) signScreen).jsmacros_setLine(i, lines.get(i));
             }
             info.cancel();
         }
     }
-    
-    
-    // IGNORE
-    public MixinClientPlayerEntity(ClientWorld world, GameProfile profile) {
-        super(world, profile);
+
+    @Inject(method = "tickMovement", at = @At(value = "INVOKE", target = "Lnet/minecraft/client/tutorial/TutorialManager;onMovement(Lnet/minecraft/client/input/Input;)V"))
+    public void overwriteInputs(CallbackInfo ci) {
```

Comment:
this might cause issues with baritone's PlayerInput class,
I was thinking overwrite player.input with your own so the player can only be controlled by the script during the movement queue; this'll also prevent user input from messing it up